marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.smart-mastering/impl/process.xqy

```diff
@@ -79,7 +79,13 @@ declare variable $STRING-TOKEN := "##";
  : @param $matches
  : @return  map(unique ID -> sequence of URIs)
  :)
-declare function proc-impl:consolidate-merges($matches as map:map) as map:map
+declare function proc-impl:consolidate-merges(
```

Comment:
adjust the consolidate-merges to pull in any other matches outside of the current batch